wdl/tasks/bedops.wdl

```diff
@@ -167,3 +167,32 @@ task merge {
         disks: resources.disks
     }
 }
+
+
+task difference {
+    input {
+        Array[File] subtrahends
+        File minuend
```

Comment:
Lol, had to look these up.

Reply:
Yeah, was pretty stoked to actually have a reason to use these :)